src/main/java/net/mcreator/ui/help/HelpLoader.java

```diff
@@ -88,23 +88,22 @@ public static String loadHelpFor(IHelpContext helpContext) {
 				if (getFromCache(helpContext.getEntry()) != null) {
 					helpString.append(renderer.render(parser.parse(getFromCache(helpContext.getEntry()))));
 				} else {
-					helpString.append("No help entry is defined for <tt>").append(helpContext.getEntry())
-							.append("</tt> yet.<br>");
+					helpString.append(L10N.t("help.loader.no_help_entry",helpContext.getEntry(), "</tt> yet.<br>"));

```

Comment:
You are still concatenating `</tt> yet.<br>` from the code, instead of moving it to the template, applies to other externalizations too.

Reply:
I don't really understand what you mean, so I tried something.

Reply:
I did not mean this. Externalized strings support replace patterns so no concatenations (string1 + string2, or StringBuilder#append) are needed, and this mechanic needs to be adapted when externalizing.

Eg.:

`"Value is" + value + "units"`

needs to be externalized as:

`key=Value is {0} units`

and call

`L10N.t("key", value)`

and not like

`L10N.t("key1") + value + L10N.t("key2")`

Reply:
Change

`helpString.append(L10N.t("help.loader.no_help_entry",helpContext.getEntry(), "</tt> yet.<br>"`

to

`helpString.append(L10N.t("help.loader.no_help_entry",helpContext.getEntry()`

and put

`</tt> yet.<br>`

inside translation file in the same key.

Reply:
Oh ok, I understand now. I was stupid because I already knew it was possible, but I forgot that. xD

Reply:
Just that we got "synced" :D